cluster/kubevirtci.sh

```diff
@@ -15,22 +15,39 @@
 export KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-'k8s-1.17'}
 
 KUBEVIRTCI_VERSION='0b941ea5dc647d3aea6f6a6fff95563d6ce0445e'
-KUBEVIRTCI_PATH="${PWD}/_kubevirtci"
+KUBEVIRTCI_REPO='https://github.com/kubevirt/kubevirtci.git'
+KUBEVIRTCI_PATH=${KUBEVIRTCI_PATH:-"${PWD}/_kubevirtci/cluster-up/"}
+KUBEVIRTCI_PARENT_PATH="$(dirname "$KUBEVIRTCI_PATH")"
```

Comment:
This is KUBEVIRTCI_PATH and the one above is CLUSTER_UP_PATH, no?

Reply:
yes, but for the same [reasoning](https://github.com/kubevirt/cluster-network-addons-operator/pull/635#discussion_r514416276)  - I thought to call it like so.